requirements-tensorflow.txt

```diff
@@ -1 +1,2 @@
 tensorflow~=2.5.2
+openvino-dev==2021.4.1
```

Comment:
Why do we have a different name for tensorflow?

Reply:
Package `openvino` is just a runtime and it can run ONNX models (PyTorch). For TensorFlow models it's required to use additional tool Model Optimizer which goes with `openvino-dev`. So `openvino-dev = MO + openvino`